runbotics/runbotics-orchestrator-ui/src/views/process/ProcessBuildView/Modeler/ConfigureActionPanel/Actions/ApiActions.ts

```diff
@@ -1,7 +1,7 @@
 import { translate } from 'src/hooks/useTranslations';
 import { IBpmnAction, Runner } from './types';
 
-const apiActions: Readonly<Record<string, IBpmnAction>> = {
+const apiActions: () => Record<string, IBpmnAction> = () => ({
```

Comment:
I think that there should be await for namespace loading

Reply:
I converted this to _async_ with await isNamespaceLoaded but now got error, because in index.ts and DefaultGroups.ts its unresolved promise instead of a function:

**DefaultGroups:**

```
export const defaultActionGroups = {
...
api: {
        label: translate('Process.Details.Modeler.ActionPanel.ActionsGroup.API'),
        items: Object.values(**apiActions()**),
    },
...
```

**index.ts line 176 - 180:**

...
    ...mailActions(),
    ...asanaActions(),
    ...googleSheetsActions(),
    ...jiraActions(),
    **...apiActions()**,
}

also got type-oriented error

- any tips?

- Is there a way to do it without creating a promise? 
- Is the _await for namespace loading_ necessary for sure?